api/mailer/SceGoogleApiHandler.js

```diff
@@ -147,6 +147,52 @@ class SceGoogleApiHandler {
       }
     });
   }
+
+  /**
+   * gabs all the events from the calendar id passed into the method,
+   * will log the next 10 events into the console.
+   * @param calendarId  calendar id specified for which calendar to pull from
+   */
+  getEventsFromCalendar(calendarId) {
+    // creates a const calendar for use in this method
+    const calendar = google.calendar({ version: 'v3', auth: this.oAuth2Client})
+    // lists the next 10 events in the current calendar
+    calendar.events.list({
+          calendarId: calendarId,
+          timeMin: (new Date()).toISOString(),
+          maxResults: 10,
+          singleEvents: true,
+          orderBy: 'startTime',
+        },
+        (err, res) => {
+          if (err) return console.log('The API returned an error: ' + err);
+          const events = res.data.items;
+          if (events.length) {
+            console.log('Upcoming 10 events:');
+            events.map((event, i) => {
+              const start = event.start.dateTime || event.start.date;
+              console.log(`${start} - ${event.summary}`);
+            });
+          } else {
+            console.log('No upcoming events found.');
+          }
+        });
+  }
+
+  // Test method, to be removed for production
+  async test() {
+    const calendarId = config.googleCalendarIds.PRIMARY_CAL;
+    const scopes = ['https://calendar.google.com/'];
+    const pathToToken = __dirname + '/../config/token.json';
+    const apiHandler = new SceGoogleApiHandler(
+        scopes, pathToToken);
+    console.log(await apiHandler
+        .getEventsFromCalendar(calendarId));
+  }
 }
 
+//new object to test the newly implemented methods
+const test = new SceGoogleApiHandler('https://calendar.google.com/',  __dirname + '/../config/token.json');
```

Comment:
remove these added lines as we can add test functions during development instead.